style.css

```diff
@@ -0,0 +1,94 @@
+body{
```

Comment:
Some minor recommendations here and below:
Before the opening brace put one space.
Read more in http://codeguide.academy/html-css.html